src/PackagePrivate/Humanizer/InternationalizedHumanizer.php

```diff
@@ -247,7 +244,55 @@ private function humanizeYearMonthDay( ?string $year, ?string $month, ?string $d
 		);
 	}
 
-	private function humanizeYear( int $year, UnspecifiedDigit $unspecifiedDigit ): string {
+	private function scaleToMessageKey( int $scale ): string {
+		switch( $scale ) {
+			case 1 : return 'edtf-year';					// X
+			case 2 : return 'edtf-decade';					// XX
+			case 3 : return 'edtf-century';					// XXX
+			case 4 : return 'edtf-millennium';				// XXXX
+			case 5 : return 'edtf-decem-millennium';		// XXXXX
+			case 6 : return 'edtf-hundreds-of-thousands';	// XXXXXX
+			case 7 : return 'edtf-million';					// XXXXXXX
+			case 8 : return 'edtf-tens-of-millions';		// XXXXXXXX
+			case 9 : return 'edtf-hundreds-of-millions';	// XXXXXXXXX
+			case 10 : return 'edtf-billion';				// XXXXXXXXXX
+			case 11 : return 'edtf-tens-of-billions';		// XXXXXXXXXXX
+			case 12 : return 'edtf-hundreds-of-billions';	// XXXXXXXXXXXX
+			case 13 : return 'edtf-trillion';				// XXXXXXXXXXXXX
+		}
+
+		// FIXME: reuse recursively the scale with trillions
+		// e.g. tens-of-trillions etc., 
+		return 'edtf-tens-of-trillions';
```

Comment:
Let's first make sure we actually need this. I am dubious that these are valid EDTF dates.

Reply:
sure, the only sensible use-case is in cosmology

Reply:
> Let's first make sure we actually need this. I am dubious that these are valid EDTF dates.

actually they are required only for testing purposes, e.g. by an user who wonder what is the output entering something like  that 56XXXXXXXXXXXXXXXXXXX